src/model/storage.py

```diff
@@ -14,57 +14,60 @@
 
 import logging
 
-from cobra.io import read_sbml_model
+import requests
+from cobra.io.dict import model_from_dict
 
 from model.app import app
 
 
 logger = logging.getLogger(__name__)
 
 
-class ModelMeta:
-    """A metabolic model, with metadata and an internal (lazy-loaded in development) cobrapy model instance."""
+class ModelWrapper:
+    """A wrapper for a cobrapy model with some additional metadata."""
 
-    def __init__(self, model_id, species, namespace, biomass_reaction):
-        self.model_id = model_id
-        self.species = species
-        self.namespace = namespace
+    def __init__(self, model, organism_id, biomass_reaction):
```

Comment:
It didn't have a docstring before but writing down the expected arguments and types would be beneficial.